cmake/std/pr_config/pullrequest.ini

```diff
@@ -0,0 +1,366 @@
+# - installation-2: Installation testing, testing phase
+#
+[MODE_MAP]
+standard: simple_testing.cmake
+installation_1: installation_testing_install.cmake
+installation_2: installation_testing_test.cmake
+
+
+
+[ENABLE_MAP]
+# Entries in the ENABLE_MAP get inserted into the files:
+# - `packageEnables.cmake`
+# - `package_subproject_list.cmake` and
+#
+# Instead of using the TriBiTS tool to determine changes.
+Trilinos_pullrequest_python_2: TrilinosFrameworkTests
+Trilinos_pullrequest_python_3: TrilinosFrameworkTests
+Trilinos_pullrequest_gcc_8.3.0_installation_testing A: Kokkos
+# Trilinos_pullrequest_gcc_8.3.0_installation_testing B: Tpetra
+
+
+
+[PR_JOB_PARAMETERS]
+# Section for global job parameters
+# max-test-parallelism is the max parallelism any given test will have.
+#     for example, if one test runs with 4 OpenMP threads, we would set
+#     this to 4.
+max-test-parallelism : 4
+
+
+
+[SEMS-ENV]
+module-purge:
+module-use: /projects/sems/modulefiles/projects
+module-load sems-env:
+
+
+
+[SEMS-DEFAULT]
+use SEMS-ENV:
+module-load sems-openmpi  : 1.10.1
+module-load sems-boost    : 1.63.0/base
+module-load sems-zlib     : 1.2.8/base
+module-load sems-hdf5     : 1.10.6/parallel
+module-load sems-netcdf   : 4.7.3/parallel
+module-load sems-parmetis : 4.0.3/parallel
+module-load sems-scotch   : 6.0.3/nopthread_64bit_parallel
+module-load sems-superlu  : 4.3/base
+module-load sems-git      : 2.10.1
+module-load sems-cmake    : 3.10.3
+setenv OMP_NUM_THREADS    : 2
+
+
+
+[ATDM-ENV]
+module-load atdm-env:
+
+
+
+[ATDM-DEFAULT]
+use ATDM-ENV:
+module-load atdm-ninja_fortran: 1.7.2
+
+
+
+[Trilinos_pullrequest_gcc_4.8.4]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 4.8.4
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+
+
+
+[Trilinos_pullrequest_gcc_4.9.3_SERIAL]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 4.9.3
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+module-remove sems-hdf5:
+module-remove sems-netcdf:
+module-remove sems-openmpi:
+module-remove sems-parmetis:
+module-remove sems-scotch:
+module-load sems-hdf5: 1.10.6/base
+module-load sems-netcdf: 4.7.3/base
+module-load sems-metis: 5.1.0/base
+setenv OMP_NUM_THREADS: 1
+
+
+
+
+[Trilinos_pullrequest_gcc_7.2.0]
+use SEMS-ENV:
+use ATDM-ENV:
+module-load sems-gcc: 7.2.0
+use SEMS-DEFAULT:
+use ATDM-DEFAULT:
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
+
+
+
+[Trilinos_pullrequest_gcc_7.2.0_debug]
+use Trilinos_pullrequest_gcc_7.2.0:
+
+
+
+[Trilinos_pullrequest_gcc_7.2.0_serial]
+use SEMS-ENV:
+module-load sems-git: 2.10.1
+module-load sems-gcc: 7.2.0
+module-load sems-python: 2.7.9
+module-load sems-boost: 1.63.0/base
+module-load sems-zlib: 1.2.8/base
+module-load sems-hdf5: 1.10.6/base
+module-load sems-netcdf: 4.7.3/base
+module-load sems-metis: 5.1.0/base
+module-load sems-superlu: 4.3/base
+module-load sems-cmake: 3.10.3
+use ATDM-ENV:
+module-load atdm-ninja_fortran: 1.7.2
+setenv SEMS_FORCE_LOCAL_COMPILER_VERSION: 4.9.3
+setenv OMP_NUM_THREADS: 2
```

Comment:
I don't understand this in relation to `Trilinos_pullrequest_gcc_7.2.0` above.  Shouldn't it just be using that one and making whatever changes are necessary?

Reply:
Nope.

The section names == the Jenkins Job name.  In this case, there are Jenkins jobs named: 
- `Trilinos_pullrequest_gcc_7.2.0`
- `Trilinos_pullrequest_gcc_7.2.0_serial`
- `Trilinos_pullrequest_gcc_7.2.0_debug`

so there must be a section for each or things will fail.  In this case, 7.2.0 and debug are identical so that one just loads in the same stuff from 7.2.0, but the _serial_ one must load the serial version of several packages. It also omits openmpi, etc.